episodes/contact-matrices.Rmd

```diff
@@ -0,0 +1,228 @@
+  symmetric = TRUE
+)
+contact_data
+```
+
+**Note: although the contact matrix is not mathematically symmetric, it satisfies the condition that the total number of contacts of one group with another is the same as the reverse. For the mathematical explanation see [here](https://epiforecasts.io/socialmixr/articles/socialmixr.html#symmetric-contact-matrices).**
+
+
+::::::::::::::::::::::::::::::::::::: callout
+### Why would a contact matrix be non-symmetric?
+
+One of the arguments of the function `contact_matrix()` is `symmetric=TRUE`. This means that the total number of contacts of age group 1 with age group 2, should be the same as the total number of contacts of age group 2 and age group 1 (see the `socialmixr` [vignette](https://cran.r-project.org/web/packages/socialmixr/vignettes/socialmixr.html) for more detail). However, when contact matrices are estimated from surveys or other sources, the *reported* number of contacts may differ by age group resulting in a non-symmetric contact matrix [(Prem et al 2021)](https://doi.org/10.1371/journal.pcbi.1009098).
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+The example above uses the POLYMOD survey. There are a number of surveys available in `socialmixr`, to list the available surveys use `list_surveys()`. To download a survey, we can use `get_survey()`
+
+```{r, message = FALSE, warning = FALSE}
+zambia_sa_survey <- get_survey("https://doi.org/10.5281/zenodo.3874675")
+```
+
+
+
+::::::::::::::::::::::::::::::::::::: challenge 
+
+## Zambia contact matrix
+
+After downloading the survey, load contact and population data for Zambia using the following age bins:
+
++ [0,15)
++ 20 +
+
+:::::::::::::::::::::::: solution 
+
+```{r polymod_poland}
+contact_data_zambia <- contact_matrix(
+  survey = zambia_sa_survey,
+  age.limits = c(0, 20),
+  symmetric = TRUE
+)
+contact_data_zambia
+```
+:::::::::::::::::::::::::::::::::
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+## Analyses with contact matrices
+
+Contact matrices can be used in a wide range of epidemiological analyses, they can be used:
+
++ to calculate the basic reproduction number while accounting for different rates of contacts between age groups [(Funk et al. 2019)](https://doi.org/10.1186/s12916-019-1413-7),
++ to calculate final size of an epidemic, see the R package `{finalsize}`,
++ to assess the impact of interventions finding the relative change between pre and post intervention contact matrices to calculate the relative difference in $R_0$ [(Jarvis et al. 2020)](https://doi.org/10.1186/s12916-020-01597-8),
++ and in mathematical models to account for group specific contact patterns (see next section.)
+
+::::::::::::::::::::::::::::::::::::: callout
+## Normalisation
+
+There are times when we need to change the scale of our contact matrices e.g. to compare matrices to model projected matrices [(Prem et al. 2021)](https://doi.org/10.1371/journal.pcbi.1009098) the entries need to be on the same scale.
```

Comment:
I think this is usually motivated by wanting to use a contact matrix as next generation matrix (NGM) with a given R0 (which is given by the dominant eigenvalue). Normalising the contact matrix means I can just use R0 * matrix to get the NGM.

Not quite sure how best to phrase this here without being overly complicated but perhaps it could be related to the equations below (which if C is normalised give a "traditonal" R0 of beta/gamma).